qmxgraph/page/utils.js

```diff
@@ -93,9 +93,7 @@ graphs.utils.obtainDecorationStyle = function obtainDecorationStyle (
 
     var rotation = graphs.utils.calculateEdgeAngle(graph, edge);
 
-    // + 90 as rotation of objects in mxGraph, or at least when using rotation
-    // by style, have 0 degrees concurrent with X axis.
-    rotation = (rotation + 90) % 360;
+    rotation = rotation % 360;
```

Comment:
I think the `+ 90` was added for some reason.
Why not change so we have a base rotation that can be defined in the individual styles?